pkg/driverbuilder/docker.go

```diff
@@ -98,7 +98,8 @@ func mustCheckArchUseQemu(ctx context.Context, b *builder.Build, cli *client.Cli
 	}
 
 	err = cli.ContainerStop(ctx, qemuImage.ID, nil)
-	if err != nil {
+	// Do not leave here if container is already stopped
+	if err != nil && !client.IsErrNotFound(err) {
```

Comment:
This is a possible fix for qemu cross builds.

Reply:
Hmm... is this required for the new `integration_test` target to work?